File: add/codesnippet/VisualBasic/t-system.windows.forms.d_99_1.vb
Private Sub DataGridView1_AutoSizeColumnsModeChanged(sender as Object, e as DataGridViewAutoSizeColumnsModeEventArgs) _ 
     Handles DataGridView1.AutoSizeColumnsModeChanged

    Dim messageBoxVB as New System.Text.StringBuilder()
    messageBoxVB.AppendFormat("{0} = {1}", "PreviousModes", e.PreviousModes)
    messageBoxVB.AppendLine()
    MessageBox.Show(messageBoxVB.ToString(),"AutoSizeColumnsModeChanged Event")

End Sub

File: add/codesnippet/CSharp/t-system.windows.forms.d_99_1.cs
private void DataGridView1_AutoSizeColumnsModeChanged(Object sender, DataGridViewAutoSizeColumnsModeEventArgs e) {

System.Text.StringBuilder messageBoxCS = new System.Text.StringBuilder();
messageBoxCS.AppendFormat("{0} = {1}", "PreviousModes", e.PreviousModes );
messageBoxCS.AppendLine();
MessageBox.Show(messageBoxCS.ToString(), "AutoSizeColumnsModeChanged Event" );
}